requirements.txt

```diff
@@ -1,6 +1,6 @@
 Flask==0.10.1
 MarkupSafe==0.23
-nltk==2.0.4
+nltk==2.0.5
```

Comment:
Ok, found https://github.com/nltk/nltk/issues/824 which explains the issue. Hopefully no other changes creeped into 2.0.5.